### DISCUSSION TITLE
v1.64.0

### DISCUSSION THREAD

**Discussion by alextran1502** [immich-app/immich#2971] [Announcements]
# v1.64.0

## Highlights

Hello, Immich fans! Welcome to the `v1.64.0` release of Immich. There are many updates in this version that we hope you'll like. Some of the key highlights include:

- Fixed issue with video progress bar interfering with action buttons and the seek mechanism - Thanks, @brighteyed, for the phenomenon fix for this.
- Guard against missing EXIF information on the Android version due to a lack of proper permission check
- Minor fixes for some raw file's acceptance criteria.
- Add more logs on mobile to help debugging some outstanding issues

- - - -

And as always, bugs are fixed, and many other improvements also come with this release.

Please consider supporting the project.

## Support

<p align="center">
<img src="https://media.giphy.com/media/LStqgGESXW8XnuCv5y/giphy.gif" width="250" title="Loading ~4000 images/videos"> 
</p>

If you find the project helpful, you can support Immich via the following channels.

- Monthly donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502)
- One-time donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502?frequency=one-time&sponsor=alextran1502)
- [Librepay](https://liberapay.com/alex.tran1502/)
- [buymeacoffee](https://www.buymeacoffee.com/altran1502)
- Bitcoin: 1FvEp6P6NM8EZEkpGUFAN2LqJ1gxusNxZX

It is a great way to let me know that you want me to continue developing and working on this project for years to come.

## What's Changed
* feat(blog): Make blog public, add June 2023 update post by @bo0tzz in https://github.com/immich-app/immich/pull/2935
* fix(deps): install poetry in pump workflow by @mertalev in https://github.com/immich-app/immich/pull/2938
* refactor(ml): modularization and styling by @mertalev in https://github.com/immich-app/immich/pull/2835
* chore(server): allow Windows' version of the MIME types for raw photos. by @elliotclee in https://github.com/immich-app/immich/pull/2945
* chore(ml): added locustfile by @mertalev in https://github.com/immich-app/immich/pull/2926
* chore(mobile): Add more error log by @alextran1502 in https://github.com/immich-app/immich/pull/2949
* add Turkish README file by @nurullahturkoglu in https://github.com/immich-app/immich/pull/2943
* chore(web): Update OSM tile URLs by @KeszeiTheOne in https://github.com/immich-app/immich/pull/2961
* feat(Android): guard against missing EXIF info by @fyfrey in https://github.com/immich-app/immich/pull/2965
* feat(mobile): custom video player controls by @brighteyed in https://github.com/immich-app/immich/pull/2960
* fix(mobile): Normalize progress bar value by @brighteyed in https://github.com/immich-app/immich/pull/2967
* feat(web): store albums sorting policy in local storage by @Ethan13310 in https://github.com/immich-app/immich/pull/2966

## New Contributors
* @nurullahturkoglu made their first contribution in https://github.com/immich-app/immich/pull/2943
* @KeszeiTheOne made their first contribution in https://github.com/immich-app/immich/pull/2961
* @Ethan13310 made their first contribution in https://github.com/immich-app/immich/pull/2966

**Full Changelog**: https://github.com/immich-app/immich/compare/v1.63.2...

<hr /><em>This discussion was created from the release <a href='https://github.com/immich-app/immich/releases/tag/v1.64.0'>v1.64.0</a>.</em>

**Comment by mkjustuk** [comment]
Hi Alex, thanks for another update, amazing how fast this project iterates.

Quick question - would it be a chore if you could mention in the release notes if the docker compose and environment files have changed since the last release? If they haven't it means I can just update the stack in Docker without checking.

Another option would be to have a version number for each in the filename and I can check that instead?

**Reply by bo0tzz** [reply]
If there are changes you need to make to the deployment, that is usually mentioned in the release notes.

**Reply by mkjustuk** [reply]
Ok, that's great.  I guess I haven't been here long enough to see that scenario then.  In which case that's perfect, thanks.
